src/style.css

```diff
@@ -1 +1,27 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars--1 > :nth-child(1),
+.stars--2 > :nth-child(-n + 2),
+.stars--3 > :nth-child(-n + 3),
+.stars--4 > :nth-child(-n + 4),
+.stars--5 > :nth-child(1n) {
```

Comment:
The CSS selectors for activating stars are incorrect. The '>' combinator is not needed here. The selectors should be like '.stars--1:nth-child(1)' without the '>' to correctly apply the styles to the first child of the '.stars--1' block. This is crucial for the functionality of the star rating system.